### ISSUE TITLE
Quick Poll - candidates with the same name

### ISSUE THREAD

**Issue by masiarek** [Equal-Vote/star-server#504]
Quick Poll - candidates with the same name?

<img width="648" alt="image" src="https://github.com/Equal-Vote/star-server/assets/857777/d280fa6f-1b28-42ec-bfad-e01ec0fe05e4">

**Comment by ArendPeter** [comment]
good point, I suppose it would make sense to have a rule that all candidate names need to be unique

**Comment by ArendPeter** [comment]
This will require updating the validation for both quick polls and when adding races through the admin panel.  

Samuel made a similar fix recently that can be used as a reference: https://github.com/Equal-Vote/star-server/pull/648

**Comment by mickaelfo** [comment]
> This will require updating the validation for both quick polls and when adding races through the admin panel.
> 
> Samuel made a similar fix recently that can be used as a reference: #648


It's fixed

![image](https://github.com/user-attachments/assets/2495c191-ebe4-46f3-9593-2f31b7ff16e6)
![image](https://github.com/user-attachments/assets/4336a4d2-e396-48a8-a5b1-9601eff91232)

**Comment by ArendPeter** [comment]
Awesome, thanks!
